service/src/test/java/uk/gov/hmcts/reform/fpl/service/DocumentServiceTest.java

```diff
@@ -0,0 +1,66 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.document.domain.Document;
+import uk.gov.hmcts.reform.fpl.model.common.DocmosisDocument;
+import uk.gov.hmcts.reform.fpl.model.docmosis.DocmosisCaseManagementOrder;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.fpl.enums.DocmosisTemplates.CMO;
+import static uk.gov.hmcts.reform.fpl.utils.DocumentManagementStoreLoader.document;
+
+@ExtendWith(SpringExtension.class)
+class DocumentServiceTest {
+
+    @Mock
+    private DocmosisDocumentGeneratorService documentGeneratorService;
+
+    @Mock
+    private UploadDocumentService uploadDocumentService;
+
+    @Captor
+    private ArgumentCaptor<String> captor;
+
+    @InjectMocks
+    private DocumentService service;
+
+    @Test
+    void shouldReturnDocumentWithDraftTitleWhenDraftBackgroundIsPresent() {
+        DocmosisCaseManagementOrder template = DocmosisCaseManagementOrder.builder()
+            .draftbackground("Present")
+            .build();
+
+        Document document = document();
+        initMocks(template, document);
+
+        assertThat(service.getDocumentFromDocmosisOrderTemplate(template, CMO)).isEqualTo(document);
+        assertThat(captor.getValue()).isEqualTo("draft-case-management-order.pdf");
+    }
+
+    @Test
+    void shouldReturnDocumentWithIssuedTitleWhenDraftBackgroundIsNull() {
+        DocmosisCaseManagementOrder template = DocmosisCaseManagementOrder.builder().build();
+        Document document = document();
```

Comment:
Document can just be declared once at the top and then not doesn't need to be passed into `initMocks

Reply:
Yes I agree, it could be!

Reply:
done!